src/codegen/pythoncg.cpp

```diff
@@ -1859,6 +1859,13 @@ void PythonTemplateParser::SetupModulePrefixes()
 	ADD_PREDEFINED_PREFIX( wxTL_3STATE, wx.dataview. );
 	ADD_PREDEFINED_PREFIX( wxTL_USER_3STATE, wx.dataview. );
 	ADD_PREDEFINED_PREFIX( wxTL_DEFAULT_STYLE, wx.dataview. );
+	ADD_PREDEFINED_PREFIX( wxTL_DEFAULT_STYLE, wx.dataview. );
+	ADD_PREDEFINED_PREFIX( wxDATAVIEW_CELL_INERT, wx.dataview. );
+	ADD_PREDEFINED_PREFIX( wxDATAVIEW_COL_HIDDEN, wx.dataview. );
+	ADD_PREDEFINED_PREFIX( wxDATAVIEW_COL_RESIZABLE, wx.dataview. );
+	ADD_PREDEFINED_PREFIX( wxDATAVIEW_COL_SORTABLE, wx.dataview. );
+	ADD_PREDEFINED_PREFIX( wxDATAVIEW_CELL_ACTIVATABLE, wx.dataview. );
+	ADD_PREDEFINED_PREFIX( wxDATAVIEW_COL_REORDERABLE, wx.dataview. );
```

Comment:
These symbols of two enums are randomly listed, i think it would be better to sort them alphabetically, this would look cleaner.